Formula/guile.rb

```diff
@@ -48,6 +49,15 @@ class Guile < Formula
     cause "Segfaults during compilation"
   end
 
+  # http://debbugs.gnu.org/cgi/bugreport.cgi?bug=23870
+  # https://github.com/Homebrew/homebrew-core/issues/1957#issuecomment-229347476
+  if MacOS.version >= :el_capitan
```

Comment:
🙇. Thanks Misty. Massively appreciate everything you've done around 10.12, especially as I've had less time to be here lately.